src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+class Square extends Figure {
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    private double side;
+
+    @Override
+    public double area() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Square, Color:" + color + ", Side: " + side + ", Area:" + area());
```

Comment:
Use 'name()' for getting String representation of enum constants. If 'color' is an enum type, you should use 'color.name()' instead of 'color.toString()'.